### DISCUSSION TITLE
Apply Form logic in Table in antd

### DISCUSSION THREAD

**Discussion by alvin-647** [ant-design/ant-design#38037] [Q&A]
Dear all, I am currently implementing a form table component based on ant Form and ant Table. My implementation is to make use of Form.List to wrap the Table component and Form.Item to wrap the cell component for editable cells.
i.e. for Form Table Component,
```
<Form.List>
{fields => (
<Table
dataSource={fields}
 {...tableProps}
/>
)}
</Form.List>
```
for EditableCell,
```
<Form.Item {...formItemProps} >
<Input />
</Form.item>
```
However, if I need to make both header cell and row cell editable, it seems this implementation cannot work, as the data model (the form list) is supposed to only read row cell value. I would like to have some inspirations. Many thanks.

**Comment by bombillazo** [comment]
Were you able to solve this? I have a use case of a Form inside a table as well.
